.gitignore

```diff
@@ -43,3 +43,5 @@ Gemfile.lock
 venv
 bears/upload/
 package-lock.json
+.pytest_cache
+
```

Comment:
Why did you add one extra blank line? 😃  We already have one blank line in .gitignore file.

Reply:
@sangamcse, it's fixed now :)

Reply:
No, you need to amend the original commit.

Only one commit per PR for a simple PR like this.

And that one commit must have a good commit messge.

Anyway ... unfortunately, this PR is no longer needed due to https://github.com/coala/coala-bears/pull/2531